README.md

```diff
@@ -98,9 +98,15 @@ Known edge cases and other topics are covered in [the FAQ](/FAQ.md).
 Contributions are welcome! Please read the [CONTRIBUTING.md](/CONTRIBUTING.md) document for details
 on getting started.
 
+## Developing
+
+Developing the JAVA SDK requires an inherent understanding. Please read the [DEVELOPING.md](/DEVELOPING.md) 
```

Comment:
> Developing the JAVA SDK requires an inherent understanding.

The phrase "inherent understanding" doesn't really say what I think you want to say. Perhaps this might be a better phrasing:

> There are portions of the SDK that are somewhat complex or fiddly.

Reply:
Noted.